sklearn/linear_model/logistic.py

```diff
@@ -605,11 +605,13 @@ def logistic_regression_path(X, y, pos_class=None, Cs=10, fit_intercept=True,
     classes = np.unique(y)
     random_state = check_random_state(random_state)
 
-    if pos_class is None and multi_class != 'multinomial':
-        if (classes.size > 2):
-            raise ValueError('To fit OvR, use the pos_class argument')
+    if classes.size <= 2:
+        multi_class = 'ovr'
         # np.unique(y) gives labels in sorted order.
-        pos_class = classes[1]
+        if pos_class is None:
```

Comment:
Took the liberty to push this if, which I thought was missing. Still some other issues with the coef_ shape.